app/views/layouts/application.html.erb

```diff
@@ -5,12 +5,15 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
```

Comment:
We would include this with the `stylesheet_link_tag` method similar to the below. You just need to replace the first param with the full URL.